### ISSUE TITLE
Any version below v.16.40.2 just doesn't open

### ISSUE THREAD

**Issue by SecretGamerV071** [BedrockLauncher/BedrockLauncher#152]
I tried to open a world in v1.14.60.5; it just didn't open after 'Registering package" There was no error, the game didn't open in the first place.

**Comment by TresAbhi** [comment]
Can you provide a log?

**Comment by SecretGamerV071** [comment]
I told you, it dosen't show UP in the log. The game just doesn't open. I click on play, it loads for a few seconds, then nothing happens. No error message, no game no nothing. I have an insider account as well.

**Comment by Rayth** [comment]
have you checked the log.txt file? also check `%LocalAppData%\CrashDumps` in case something is there

**Comment by TresAbhi** [comment]
There should be a log anyway.

**Comment by SecretGamerV071** [comment]
![image](https://user-images.githubusercontent.com/95269740/154076969-1302ac68-7dad-42b7-8032-43fde3df4759.png)

**Comment by SecretGamerV071** [comment]
> There should be a log anyway.

Where? I can't find it in the .minecraft_bedrock folder

**Comment by SecretGamerV071** [comment]
Okay. So I think I figured out the issue. I had not noticed it before, but, because I joined beta, Windows replaced my "normal" Minecraft app, which would have shown the ugly beta panorama on startup, with the new "Minecraft Preview" app. Maybe that's why it isn't recognizing my game. The problem is: if I want to solve this issue, I'd need to leave the Beta programme. So you guys need to make the Bedrock Launcher copatible with Minecraft Preview
